docs/components.md

```diff
@@ -0,0 +1,41 @@
+# Components
+This page provides a brief description of each KubeSaw component.
+
+## Host Operator
+The host-operator is _KubeSaw's control-plane_, is responsible for managing the users, spaces, notifications and monitoring the kubesaw instance.
```

Comment:
```suggestion
The host-operator is _KubeSaw's control-plane_, and is responsible for managing the users, spaces, notifications and monitoring the kubesaw instance.
```

Reply:
or you can take rafaela suggestion and make it even in both operators

Reply:
yes let's go with that if you agree, since I've already applied that before seeing your comment :)

Reply:
pls see https://github.com/kubesaw/kubesaw.github.io/pull/3/commits/fbf8a46fd9c56a396e1267059d20b72a34816728